samples/animate-images-with-image-overlay/src/main/java/com/esri/arcgismaps/sample/animateimageswithimageoverlay/screens/AnimateImagesWithImageOverlayScreen.kt

```diff
@@ -0,0 +1,209 @@
+                            title = messageTitle,
+                            description = messageDescription,
+                            onDismissRequest = ::dismissDialog
+                        )
+                    }
+                }
+            }
+        })
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun ImageOverlayMenu(
+    isStarted: Boolean,
+    fps: Int,
+    opacity: Float,
+    fpsOptions: List<Int>,
+    updateFpsOption: KFunction1<Int, Unit>,
+    updateOpacity: KFunction1<Float, Unit>,
+    updateIsStarted: KFunction1<Boolean, Unit>
+) {
+
+    var selectedFps by remember { mutableIntStateOf(60) }
+
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(start = 12.dp, end = 12.dp, bottom = 12.dp),
+        horizontalArrangement = Arrangement.Absolute.SpaceBetween
+    ) {
+        Text("Opacity:")
+        Text(text = (opacity * 100).roundToInt().toString() + "%")
+    }
+    // Opacity Slider
+    Slider(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(start = 24.dp, end = 24.dp, bottom = 12.dp),
+        value = opacity,
+        onValueChange = { updateOpacity(it) },
+        valueRange = 0f..1.0f
+    )
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(start = 12.dp, end = 12.dp, bottom = 12.dp),
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        // Start/Stop Button
+        Button(onClick = {
+            updateIsStarted(!isStarted)
+        }) {
+            Text(if (isStarted) "Stop" else "Start")
+        }
+        // FPS Dropdown Menu
+        var expanded by remember { mutableStateOf(false) }
+        ExposedDropdownMenuBox(
+            modifier = Modifier.width(150.dp),
+            expanded = expanded,
+            onExpandedChange = { expanded = !expanded }
+        ) {
+            TextField(
+                value = "$fps fps",
+                onValueChange = {},
+                readOnly = true,
+                trailingIcon = { ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded) },
+                modifier = Modifier.menuAnchor(type = MenuAnchorType.PrimaryNotEditable)
+            )
+            ExposedDropdownMenu(
+                expanded = expanded,
+                onDismissRequest = { expanded = false }
+            ) {
+                fpsOptions.forEachIndexed { index, fps ->
+                    DropdownMenuItem(
+                        text = { Text("$fps fps") },
+                        onClick = {
+                            updateFpsOption(index)
+                            selectedFps = fps
+                            expanded = false
+                        })
+                    // Show a divider between dropdown menu options
+                    if (index < fpsOptions.lastIndex) {
+                        HorizontalDivider()
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
Could use a `@Preview` function here: 
```suggestion
}


@Preview(showBackground = true)
@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES, showBackground = true)
@Composable
fun ClusterInfoContentPreview() {
    SampleAppTheme {
        Surface {
            ImageOverlayMenu(
                isStarted = true,
                fps = 60,
                opacity = 1F,
                fpsOptions = listOf(60, 30, 15),
                onFpsOptionSelected = { },
                onOpacityChanged = { },
                onIsStartedChanged = {}
            )
        }
    }
}

```